tests/integration/annotation_import/test_data_types.py

```diff
@@ -153,16 +188,7 @@ def test_import_label_annotations(client, configured_project,
     data_row.delete()
 
 
-@pytest.mark.parametrize('data_type, data_class, annotations', [
```

Comment:
why moved this one?

Reply:
The two tests have the same parameters so I extracted them out to a variable.